zapisy/apps/defect/views.py

```diff
@@ -142,6 +139,13 @@ def edit_defect_post_request(request, defect_id):
                   place=form_data['place'])
     formset.save()
 
+    if request.user.id != defect.get().reporter.id:
```

Comment:
W pracy by mi napisali, że powtarzam kod.

Reply:
Jednak wiem co zrobiłem z vue, więc nie wymagam zmiany xD

Reply:
No nie chciało mi się pisać funkcji z jedym ifem xd to dzięki :smile: